src/documentation/doc-util/doc-search.ts

```diff
@@ -69,74 +66,4 @@ ${await printDfGraphForCode(shell, code, { showCode: false, switchCodeAndGraph:
 ${collapseResult ? '</details>' : ''}
 
 	`;
-
```

Comment:
This duplicate code segment from `doc-query.ts` seems to be completely unused.
It was introduced alongside this file in #1175 🤔

Reply:
Unsure why this existed, thanks for removing if unused :D